### ISSUE TITLE
Add announcement about PicoDSP board

### ISSUE THREAD

**Issue by DatanoiseTV** [DatanoiseTV/RP2040-DSP-FreeRTOS-Template#5]
![image](https://user-images.githubusercontent.com/6614616/175049934-fc095e37-27aa-4561-8ab7-ec2b04b83dc6.png)

**Comment by TobiasVanDyk** [comment]
Yes please if that means we can have USB Audio at a 32bit resolution I will buy you a beer....

**Comment by TobiasVanDyk** [comment]
Great will try it over the weekend..

**Comment by maks** [comment]
@DatanoiseTV would you planning on making/selling these little boards? I'd be interested in these rather then the eurorack sized boards as I've got a project in mind to put a pico board inside existing midi controllers.

**Comment by DatanoiseTV** [comment]
@maks Yes, I am planning to make more boards after 07.10.

**Comment by DatanoiseTV** [comment]
@maks Let me know if you are interested, I am taking preorders now. There will be also a PicoDSP+, which will have an on-board 8 channel 12-bit ADC.

**Comment by DatanoiseTV** [comment]
![image](https://user-images.githubusercontent.com/6614616/196260086-bd8c0e53-0458-4aa2-a906-b84bf049340a.png)

**Comment by maks** [comment]
@DatanoiseTV thats excellent news! The PicoDSP+ sounds great too. Yes I'd definitely like to order some, whats the best way I can get in touch?

**Comment by DatanoiseTV** [comment]
> @DatanoiseTV thats excellent news! The PicoDSP+ sounds great too. Yes I'd definitely like to order some, whats the best way I can get in touch?

Mail is the best right now: datanoise@mailbox.org
There will be a discord coming later on.

**Comment by DatanoiseTV** [comment]
@maks Hardware Related Updates moved to https://github.com/DatanoiseTV/PicoDSP-Hardware#news-as-of-20-10-2022---introducing-the-picoadk

**Comment by maks** [comment]
Thanks @DatanoiseTV , I also emailed you a few days ago, hopefully the email got through to you?

**Comment by DatanoiseTV** [comment]
@maks unfortunately it didn't. Can you retry with syso(at)datanoise(org) please?

**Comment by maks** [comment]
@DatanoiseTV  oh, thanks for letting me know, I've tried again with this new email address.

**Comment by DatanoiseTV** [comment]
https://github.com/DatanoiseTV/PicoADK-Hardware
